ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -186,7 +191,7 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
     const emailList = emailInput.split(/[,\n]+/);
     const rtTierConfig = {
       ...getRegisteredTierConfig(this.state.institution),
-      emailAddresses: emailList.map(email => email.trim()).filter((item) => item !== '')
```

Comment:
Why did this need to change?

Reply:
It is expected to leave empty string here, that is how you can type enter to go to new line in textbox.
The empty string is trimed in the method I modified above.